general-concepts/licenses/text.xml

```diff
@@ -82,17 +82,39 @@ versions are specified elsewhere.
 </p>
 
 <p>
-Please watch for license conflicts. If the license indicated
-by the package is incompatible with the licenses used by its sources
-(e.g. BSD/MIT package including GPL sources), please contact
-the licenses team for guidance. Do not add packages that seem
-to include license term violations.
+If an installed file is built from sources using different licenses, all those
+licenses must be compatible. The effective license for the file will be the most
+restrictive of input licenses. For example:
+</p>
+
+<ol>
+  <li>
+    If a program includes code that is partially covered by GPL-2
+    and partially by the BSD license, the license for the resulting
```

Comment:
Maybe a clarification like "3-clause BSD license" would be helpful here. The original BSD license (which we list as BSD-4) is not GPL compatible.

Reply:
Well, I meant to use terms from LICENSE but sure.